lib/msal-core/src/AuthResponse.ts

```diff
@@ -4,14 +4,98 @@ import { IdToken } from "./IdToken";
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-export type AuthResponse = {
-    uniqueId: string;
-    tenantId: string;
-    tokenType: string;
-    idToken: IdToken;
-    accessToken: object;
-    scopes: Array<string>;
-    expiresIn: string;
-    account: User;
-    userState: string;
-};
+export class AuthResponse  {
```

Comment:
what if instead of a class, with no need for a constructor and a bunch of getters and setters with no tranformations, we did a type?

Reply:
I had a type before. I changed to class to make initialization of the response object simpler. We can also do any response-based logic in this class in the future, as well as set defaults for values if needed. I am open to using a type, but this was more for future changes as well as code readability.